src/ocean/time/Time.d

```diff
@@ -697,6 +703,23 @@ struct Time
                 return TimeSpan(ticks_ - epoch1970.ticks_);
         }
 
+        /**********************************************************************
+
+            Constructs a Time instance from the Unix time (time since 1/1/1970).
+
+            Params:
+                unix_time = number of seconds since 1/1/1970
+
+            Returns:
+                Time instance corresponding the given unix time.
+
+        ***********************************************************************/
+
+        static Time fromUnixTime (long unix_time)
```

Comment:
Maybe use `time_t`? This method is useful for interfacing with POSIX.

Reply:
Ah, good point, I have no idea how I missed that.

Reply:
Updated.